src/assets/locales/fr.json

```diff
@@ -284,7 +284,7 @@
   "more-menu.tos": "Conditions d'utilisation",
   "more-menu.unfavorite-room": "Supprimer la salle des favoris",
   "more-menu.whats-new": "Quoi de neuf",
-  "oauth-screen.description": "Vous devrez vous connecter pour {provider} accéder à la salle.",
+  "oauth-screen.description": "Vous devrez vous connecter à {provider} pour accéder à la salle.",
```

Comment:
I actually changed it again in https://github.com/mozilla/hubs/pull/5002/commits/5a43021cac0a58b37f433d2e0d35c5a5ff6cc5c3
"Vous devrez vous connecter par {provider} pour accéder à la salle."

FYI we have another translation here so this matches:
"oauth-screen.sign-in-button": "Se connecter par {provider}"